mint.json

```diff
@@ -145,72 +145,52 @@
     },
     {
       "group": "GETTING STARTED",
-      "pages": ["guides/overview", "guides/network-coverage"]
-    },
-    {
-      "group": "QUICKSTART: WALLET ABSTRACTION",
       "pages": [
-        "guides/quickstart/intro",
-        "guides/quickstart/dashboard",
-        {
-          "group": "Wallet-as-a-Service",
-          "pages": [
-            "guides/quickstart/waas/understanding-waas",
-            "guides/quickstart/waas/web-quickstart",
-            "guides/quickstart/waas/mobile-quickstart"
-          ]
-        },
-        {
-          "group": "Account Abstraction",
-          "pages": [
-            "guides/quickstart/aa/web-aa",
-            "guides/quickstart/aa/mobile-aa",
-            "guides/quickstart/aa/bundler",
-            "guides/quickstart/aa/paymaster"
-          ]
-        },
+        "guides/overview", 
+        "guides/network-coverage", 
+        "guides/dashboard" ,
         {
-          "group": "Connection Kits",
+          "group": "Platform Coverage",
           "pages": [
-            "guides/quickstart/connection-kits/particle-connect",
-            "guides/quickstart/connection-kits/rainbow",
-            "guides/quickstart/connection-kits/web3modal",
-            "guides/quickstart/connection-kits/web3-onboard",
-            "guides/quickstart/connection-kits/solana"
+            "guides/integrations/platform/web",
+            "guides/integrations/platform/android",
+            "guides/integrations/platform/ios",
+            "guides/integrations/platform/unity",
+            "guides/integrations/platform/unreal",
+            "guides/integrations/platform/cocos",
+            "guides/integrations/platform/flutter",
+            "guides/integrations/platform/react"
           ]
         }
       ]
     },
     {
-      "group": "UNDERSTANDING WALLET ABSTRACTION",
+      "group": "WALLET-AS-A-SERVICE",
```

Comment:
Changed structure to:
- Getting Started (Overview, Network Coverage, Dashboard, Platform Coverage)
- Wallet-as-a-Service (Quickstarts, Understanding WaaS, Product Overviews, Connection Kits, Building with Particle's Partners)
- Account Abstraction (Quickstarts, Product Overviews)
- Configuration (Appearance, Authentication)

Clearer this way and "Configuration" fits in better.